src/screens/Booking/Booking.tsx

```diff
@@ -0,0 +1,53 @@
+// @ts-nocheck
+
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import {RouteProp, useRoute} from '@react-navigation/native';
+import {FlightData} from '@components';
+import Baggage from './components/Baggage/Baggage';
+import Form from './components/PassengerForm/Form';
+import styles from './Booking.styles';
+import {RootStackParamsList} from '@navigation/types';
+
+const Booking = () => {
+  const route = useRoute<RouteProp<RootStackParamsList, 'BookingScreen'>>();
+  const [state, setState] = React.useState({
+    cabin: true,
+    checked: true,
+  });
+
+  const {
+    id,
+    destination,
+    destinationIata,
+    origin,
+    originIata,
+    time,
+  } = route.params.modalData;
+
+  const setCabinVal = () => setState({...state, cabin: !state.cabin});
+  const setCheckedVal = () => setState({...state, checked: !state.checked});
+
+  return (
+    <View style={styles.container}>
```

Comment:
Is this `View` needed? Can't the `ScrollView` be the top wrapper for this component? I'm not a fan of having nested `View` components if there's no actual need for it. 🤔
(That doesn't mean that I'm right in this case, though)

If the styling's an issue, maybe it could be fixed adding it to the `contentContainerStyle` prop of the `ScrollView`. It's not exactly the same, because it wouldn't be the parent of the ScrollView but the child of it... I'm not 100% sure, I'd need to try it out, but there should be a way to make it work as expected.